pkg/virt-launcher/virtwrap/network/podinterface.go

```diff
@@ -867,8 +873,8 @@ func (b *MasqueradeBindMechanism) generateDomainIfaceSpec() api.Interface {
 			Managed: "no",
 		},
 	}
-	if b.dhcpConfig.MAC != nil {
-		domainIface.MAC = &api.MAC{MAC: b.dhcpConfig.MAC.String()}
+	if b.mac != nil {
+		domainIface.MAC = &api.MAC{MAC: b.mac.String()}
```

Comment:
Why on the bridge binding `generateDomainIfaceSpec` you still rely on the `dhcpConfig`?